tools/runners/run-proxy.js

```diff
@@ -23,6 +26,29 @@ var Proxy = function (options) {
   self.server = null;
 };
 
+const removeExistingSocketFile = (socketPath) => {
```

Comment:
This function was copied from elsewhere. Is there no way to reuse it? At least we could say in a comment where did it originate from.